src/crc32c.c

```diff
@@ -114,14 +109,15 @@ static uint32_t crc32c_sse42(uint32_t crc, const char *buf, size_t len)
         uintptr_t crc2 = 0;
         const char *end = buf + LONG;
         do {
-            __asm__(CRC32_PTR "\t" "(%3), %0\n\t"
-                    CRC32_PTR "\t" LONGx1 "(%3), %1\n\t"
-                    CRC32_PTR "\t" LONGx2 "(%3), %2"
-                    : "+r"(crc0), "+r"(crc1), "+r"(crc2)
-                    : "r"(buf),
-                      "m"(* (const char (*)[sizeof(void*)]) &buf[0]),
-                      "m"(* (const char (*)[sizeof(void*)]) &buf[LONG]),
-                      "m"(* (const char (*)[sizeof(void*)]) &buf[LONG*2]));
+            __asm__(CRC32_PTR "\t(%1), %0"
```

Comment:
```suggestion
            __asm__(CRC32_PTR "\t%1, %0"
```
(everywhere, as the () means to load the memory, which isn't legal)

Reply:
Maybe I should just change it back to be more like what it was before:
```c
_asm__(CRC32_PTR "\t(%1), %0"
               : "+r"(crc0)
               : "r"(buf));
__asm__(CRC32_PTR "\t" LONGx1 "(%3), %1"
                    : "+r"(crc1)
                    : "m"(* (const char (*)[sizeof(void*)]) &buf[0]));
```
etc.

Reply:
anyway, I changed as you suggested